airbyte-webapp/src/components/base/TextInputContainer/TextInputContainer.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from "classnames";
+import { useState } from "react";
+
+import styles from "./TextInputContainer.module.scss";
+
+export interface TextInputContainerProps {
+  disabled?: boolean;
+  light?: boolean;
+  error?: boolean;
+  onFocus?: React.FocusEventHandler<HTMLDivElement>;
+  onBlur?: React.FocusEventHandler<HTMLDivElement>;
+}
+
+export const TextInputContainer: React.FC<TextInputContainerProps> = ({
```

Comment:
Are you planning to extend this component's usage? Because the secret text area only uses some of these props and it looks like it isn't used elsewhere.

Reply:
Yes, this will eventually be shared by the input component but the change will be done separately.